### ISSUE TITLE
Resources: Elements must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by vh8154** [pulibrary/DSS#540]
### Steps to recreate
Use axe devtools on /resources

### URL
localhost:3000/resources

### Issues
color contrast

### Severity
AA

### WCAG Success Criterion
[1.4.3 Contrast (minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

### WCAG Conformance level (A, AA, AAA)
AA

### User impact
Some people cannot read faint text. For all consumers of visual content, adequate light-dark contrast is needed between the relative luminance of text and its background for good readability. 

### Recommended fix
Element has insufficient color contrast of 1.21 (foreground color: #ebe9e2, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Change to expected contrast ratio of 4.5:1

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes
Related to #526

**Comment by rladdusaw** [comment]
This element is the next/previous pagination buttons which are implemented as links instead of button elements. Unfortunately, this html comes from a jquery data table library and can not be edited without making a fork of the repo or engaging in some javascript shenanigans. 

The better alternative would be to rewrite the data tables with something more current, but that would be substantially more effort.

**Comment by kevinreiss** [comment]
Block this ticket by something replace jquery data-tables.
